combat-trainer.lic

```diff
@@ -3407,7 +3409,7 @@ class AttackProcess
   end
 
   def dance(game_state)
-    if game_state.npcs.empty?
+    unless game_state.can_engage?
```

Comment:
Minor, but I'd prefer to swap the two branches so we can use `if` instead of `unless` here

Reply:
Sorry following up on this one. What do you mean? I'll fix the code and get this merged as there's interest from a couple others.

Reply:
```
if game_state.can_engage?
    line 3415...
else
    pause 1
end
```

Reply:
@rpherbig Is it good now?